.github/workflows/nightfalldlp.yml

```diff
@@ -23,13 +23,22 @@ jobs:
           restore-keys: |
```

Comment:
it might be good to just update this entire file to https://github.com/nightfallai/nightfall_dlp_action/blob/445a89d0ea4f1042c290e56c50c3228714d33860/.github/workflows/nightfalldlp.yml